controllers/orders/_status_name_column.htm

```diff
@@ -0,0 +1,8 @@
+<?php
+
+/** @var \Lovata\OrdersShopaholic\Models\Order $record */
+	
+?>
+<div class="shopaholic-badge"<?php if ($record->status->color): ?> style="background-color: <?= e($record->status->color) ?>"<?php endif; ?>>
```

Comment:
Inline style? Why?

Reply:
The fact is that it is difficult to predict which colors of the colorpicker the user will use in his statuses so that classes can be prepared for them.

I will be grateful for the tip

Reply:
Is there some developer guide for such case in the oficial documentation? @kharanenka?